www/js/serv-storage.js

```diff
@@ -69,23 +70,22 @@ angular.module('starter.services', [])
       temp = [];
       localStorage.setItem(tempMemAdd, JSON.stringify(temp));
 
-      API.POST('/members/import', sending, localStorage.getItem("token"))
-      .then(
-        function (response) {
-          if (response.results.errors === 0) {
+      API.addMembers(sending).then(
+        function(response) {
+          if (response.data.results.errors === 0) {
             sent = JSON.parse(localStorage.getItem(sentMemAdd)) || [];
             sent = sent.concat(sending);
             localStorage.setItem(sentMemAdd, JSON.stringify(sent));
 
             sending = [];
             localStorage.setItem(sendingMemAdd, JSON.stringify(sending));
-            return true;
+            deferred.resolve(200);
           }
           else {
             dirty = JSON.parse(localStorage.getItem(dirtyMemAdd)) || dirty;
-            for (var i = response.results.error_checksums.length - 1; i >= 0; i--) {
+            for (var i = response.data.results.error_checksums.length - 1; i >= 0; i--) {
```

Comment:
IDK what's going on here but surely using nice functions like forEach, filter map etc. will make this a lot easier to read